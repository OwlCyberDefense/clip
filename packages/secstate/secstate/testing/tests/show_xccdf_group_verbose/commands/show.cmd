/usr/sbin/secstate show PassComp PassComp-G-2-2 -v
