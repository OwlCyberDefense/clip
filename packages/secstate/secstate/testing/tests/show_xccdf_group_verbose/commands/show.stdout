PassComp-G-2-2:
	Title:  'Password'
	Description:  Group containing all groups and rules pertaining to passwords.
	Selected:  True
	Member of PassComp
	Sub Elements:
		PassComp-G-2-3
